.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Thanks SO MUCH for the pull request!
One of the things we try to do on the Monsanto projects is add comments to the files changed immediately after creating a pull request that attempt to anticipate questions reviewers will ask about the changes.
For example, for the changes you made to .babelrc, you could add a comment explaining why you needed the es2016 and react presets. What issues did adding those address?